chains/manager.go

```diff
@@ -878,6 +878,11 @@ func (m *manager) createAvalancheChain(
 		Validators:    vdrs,
 		Params:        consensusParams,
 		Consensus:     snowmanConsensus,
+
+		// block backfilling stuff
+		Peers:                          connectedBeacons,
+		AncestorsMaxContainersSent:     m.BootstrapAncestorsMaxContainersSent,
+		AncestorsMaxContainersReceived: m.BootstrapAncestorsMaxContainersReceived,
```

Comment:
will be moved to a separate config once I repackage block backfilling stuff into an ad-hoc struct